samples/fabric/fabric-cli/src/commands/asset/transfer/claim.ts

```diff
@@ -0,0 +1,226 @@
+/*
+ * Copyright IBM Corp. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { GluegunCommand } from 'gluegun'
+import * as path from 'path'
+import {
+    commandHelp,
+    pledgeAsset,
+    getNetworkConfig,
+    getLocalAssetPledgeDetails,
+    getChaincodeConfig,
+    handlePromise,
+    generateViewAddressFromRemoteConfig,
+    interopHelper
+} from '../../../helpers/helpers'
+import {
+    fabricHelper,
+    getUserCertBase64
+} from '../../../helpers/fabric-functions'
+
+import logger from '../../../helpers/logger'
+import * as dotenv from 'dotenv'
+dotenv.config({ path: path.resolve(__dirname, '../../../.env') })
```

Comment:
It'd be good to insert a blank link above this.

Reply:
It's like this in every fabric-cli file which requires `.env`...

Reply:
OK, leave it as is.